README.md

```diff
@@ -345,6 +345,24 @@ included by default unless you choose which one to exclude with the
 then you have to specify each backend that you want individually. You can get
 the full list of these flags with the `--help` flag.
 
+#### --auto-config-uri
+This is a special URI which if set, will try and pull a config from that
+location on startup. It will use the cookie file stored at
+`--auto-config-cookie-path` if specified. If successful, it will check if the
+config is different from what is currently stored. If so then it will validate
+if it is a valid json config. If so it will replace (overwrite!) the current
+config and then run with that!
+
+For example: `--auto-config-uri 'https://example.com/config.json'`.
+
+#### --auto-config-cookie-path
```

Comment:
We are using `cookie-path` except that this is the `--auto-config` prepended version =D

Reply:
I don't think that answers my question. Can you consider my comment more carefully?

Reply:
I don't see any reason for needing more than one cookie path. If we do, then we would want to call it `--for-other-specific-reason-cookie-path`. If you see the need for an alternate cookie path, please let me know and we can consider renaming this early. Since I don't see one at the moment, I will resolve this comment. Please comment again if you'd like to discuss more. Thank you.

Reply:
As neither of us can imagine needing more than one cookie path, I suggest the option be called `--cookie-path`. As it's a plain text file format, it's reasonable to require someone to concatenate multiple cookies into one file if they come up with a reason we couldn't anticipate.

It would become cumbersome and confusing to have multiple `--<foo>-cookie-path` variables for different purposes.

Reply:
We could also simply allow multiple `--cookie-path` arguments (as cURL does).